about.html

```diff
@@ -0,0 +1,187 @@
+                        <!-- <li><img class="soc-logo" src="images/Team/logos/linkedin.png"> <a href=" --link here-- "> --link here -- </a></li> -->
+                        <li><img class="soc-logo" src="images/Team/logos/gmail.png"> <a href="mike.padiernos@gmail.com">mike.padiernos@gmail.com</a></li>
+                    </ul>  
+                </div>
+                
+                <!-- Donald Hager -->
+                <div class="team-card">
+                    <div class="tCard-top">
+                        <img class="member-img" src="images/Team/martinLehman.png">
+                        <div class="member-name">    
+                            <h1>Donald Hager</h1>
+                            <h2>Full Stack</h2>
+                        </div>
+                    </div>
+                    <ul class="soc-media">
+                        <li><img class="soc-logo" src="images/Team/logos/github.png"> <a href="https://github.com/b1zz3">github.com/b1zz3</a></li>
+                        <!-- <li><img class="soc-logo" src="images/Team/logos/twitter.png"> <a href=" --link here-- "> --link here -- </a></li> -->
+                        <!-- <li><img class="soc-logo" src="images/Team/logos/linkedin.png"> <a href=" --link here-- "> --link here -- </a></li> -->
+                        <li><img class="soc-logo" src="images/Team/logos/gmail.png"> <a href="donhager23@gmail.com">donhager23@gmail.com</a></li>
+                    </ul>  
+                </div>
+
+                <!-- Christopher Quinn -->
+                <div class="team-card">
+                    <div class="tCard-top">
+                        <img class="member-img" src="images/Team/martinLehman.png">
+                        <div class="member-name">    
+                            <h1>Christopher Quinn</h1>
+                            <h2>Full Stack</h2>
+                        </div>
+                    </div>
+                    <ul class="soc-media">
+                        <li><img class="soc-logo" src="images/Team/logos/github.png"> <a href="https://github.com/christopherc1331">github.com/christopherc1331</a></li>
+                        <li><img class="soc-logo" src="images/Team/logos/twitter.png"> <a href="https://twitter.com/ChrisCruzes">twitter.com/ChrisCruzes</a></li>
+                        <li><img class="soc-logo" src="images/Team/logos/linkedin.png"> <a href="https://www.linkedin.com/in/christopher-cruz-5b7979186/">linkedin.com/ChrisCruzes</a></li>
+                    </ul>  
+                </div>
+
+
+            </section>
+
+        </div>
+
+    </body>
+
+    <footer id="footer" class="footer">
+        <p>© Copyright D'Salty Dogs of LS</p>
+        <p>Social Media Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></p>
+    </footer>
+
+</html>
```

Comment:
I don't think this is an issue, but Github has it marked in red so it might be something

Reply:
I'm not sure about this one either, i'll just get rid of the link and leave the text credit.

Reply:
I think github is assuming the "www.flaticon.com" between the a tag is a broken link. Maybe replace it with just Flaticon.